core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -139,20 +139,20 @@ private[spark] class Executor(
   private val executorPlugins: Seq[ExecutorPlugin] = {
     val pluginNames = conf.get(EXECUTOR_PLUGINS)
     if (pluginNames.nonEmpty) {
-      logDebug(s"Initializing the following plugins: ${pluginNames.mkString(", ")}")
+      logInfo(s"Initializing the following plugins: ${pluginNames.mkString(", ")}")
```

Comment:
Hi, @iRakson . At the PR description, could you describe your reason why this should be `INFO`?

Reply:
It sounds reasonable to use logInfo for this message. The message is only shown once for each executor. Debug level might be too low here.